cluster/os-3.9.0/README.md

```diff
@@ -28,10 +24,32 @@ node01    Ready     master    1h        v1.9.1+a0ce1bc657
 node02    Ready     <none>    46s       v1.9.1+a0ce1bc657
 ```
 
+## OpenShift Web Console
+
+If you want to get access to OpenShift web console you will need to add line to `/etc/hosts`
+```bash
+echo "127.0.0.1 node01" >> /etc/hosts
+```
+
+and run your cluster like this:
+
+```bash
+export KUBEVIRT_PROVIDER=os-3.9.0
+export KUBEVIRT_NUM_NODES=1 # master + one nodes
+export KUBEVIRT_PROVIDER_EXTRA_ARGS="--ocp-port 8443"
```

Comment:
Why wouldn't we default this --ocp-port arg for the user regardless?

Reply:
The only reason are port conflicts. If you don't specify the port here,  node01:8443 will exposed on a random port on a cluster, which is not used.